packages/calcite-components/src/components/dropdown/dropdown.tsx

```diff
@@ -480,11 +480,14 @@ export class Dropdown
     }
 
     this.reposition(true);
-    const maxScrollerHeight = this.getMaxScrollerHeight();
-    scrollerEl.style.maxHeight = maxScrollerHeight > 0 ? `${maxScrollerHeight}px` : "";
+    scrollerEl.style.maxBlockSize = this.getMaxHeightStyle(this.getMaxScrollerHeight());
     this.reposition(true);
   }
 
+  private getMaxHeightStyle(maxHeight: number): string {
+    return maxHeight > 0 ? `min(${maxHeight}px, 90vh)` : "";
```

Comment:
Our menu style has this:

<img width="282" alt="image" src="https://github.com/user-attachments/assets/be03f64e-92a5-4fd0-be95-680323e9b859" />

Should we make this a calcite global var or something? so we can use/reference it?

Reply:
90vh is fine if the dropdown is at the top of the page but if its centered then 45 might be better. No strong opinion here though.

Reply:
> Should we make this a calcite global var or something? so we can use/reference it?

Yeah, I'll add a variable of some sort. #sharingiscaring

> 90vh is fine if the dropdown is at the top of the page but if its centered then 45 might be better. No strong opinion here though.

I'll play with those options.